client/src/components/base/Button.tsx

```diff
@@ -7,21 +9,28 @@ export enum ButtonVariants {
 
 interface ButtonProps {
   onClick?: () => void;
+  path?: string;
   children: React.ReactNode;
   variant: ButtonVariants;
   disabled?: boolean;
   styles?: string[];
 }
 
-function Button({ onClick, children, variant, disabled, styles }: ButtonProps) {
+function Button({ onClick, path, children, variant, disabled, styles }: ButtonProps) {
+  const navigate = useNavigate();
+
+  if (onClick === undefined && path !== undefined) {
+    onClick = () => navigate(path);
+  }
+
   return (
     <button
       disabled={disabled}
       onClick={onClick}
       className={`base-btn ${variant} ${styles?.join(" ")}`}
```

Comment:
If you add `flex` to the default classNames it should align the content horizontally